omniledger/service/struct.go

```diff
@@ -42,11 +45,42 @@ func (r *roCollection) Get(key []byte) collection.Getter {
 	return r.c.Get(key)
 }
 
+func (r *roCollection) GetValues(key []byte) (value []byte, contractID string, err error) {
```

Comment:
Why is no godoc required here? Whats the diff btw get getvalues and getvalue? Am not really a fan of big interfaces.....

Reply:
As it's defined in the interface, it doesn't seem to be needed here. Perhaps I got the java-virus? Want me to remove it?

Reply:
Either the docs will convince me why it's there, or you should remove it. :)

Reply:
kept `GetValues` instead of `GetValue` and `GetContractID`.